src/main/kotlin/Car.kt

```diff
@@ -0,0 +1,22 @@
+import java.lang.IllegalArgumentException
+
+class Car {
+    private var position: Int = 0
+
+    fun accelerate(random: Int): Int {
+        if (random < 0) {
+            throw IllegalArgumentException("0보다 작은 값은 사용이 불가능합니다. [input : $random]")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(random > 0) { "0보다 작은 값은 사용이 불가능합니다. [input : $random]" }
```

Reply:
[5e30ace](https://github.com/next-step/kotlin-racingcar/pull/80/commits/5e30ace3f242a6e66a7bc8f8585d25c17ff61717) 수정 반영 하였습니다.